runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -32,10 +32,33 @@
 @ConfigGroup("loottracker")
 public interface LootTrackerConfig extends Config
 {
+	@ConfigItem(
```

Comment:
I dont think this needs a toggle?

Reply:
Some people may not want to persist data between sessions so i figured why store their data for no reason? I can revert this config change if you'd like.

Reply:
Well, we don't have toggle in master, so why add it now. And for people that doont want to see the persisted data, they will just disable the other toggle

Reply:
The changes in master were meant to just test the API performance so we didn't want to announce it when we were still testing the feasibility.

Either way I've removed that config option as requested.